src/commands/dao_cmds/token.js

```diff
@@ -0,0 +1,151 @@
+const TaskList = require('listr')
+const { ensureWeb3 } = require('../../helpers/web3-fallback')
+const { getContract } = require('../../util')
+const listrOpts = require('../../helpers/listr-options')
+const chalk = require('chalk')
+
+const ZERO_ADDR = '0x0000000000000000000000000000000000000000'
+
+exports.command =
+  'token <token-name> <symbol> [decimal-units] [transfer-enabled]'
+
+exports.describe = 'Create a new minime token'
+
+exports.builder = yargs => {
+  return yargs
+    .positional('token-name', {
+      description: 'Full name of the new Token',
+    })
+    .positional('symbol', {
+      description: 'Symbol of the new Token',
+    })
+    .option('transfer-enabled', {
+      description: 'Whether the new token will have transfers enabled',
+      default: true,
+    })
+    .option('decimal-units', {
+      description: 'Total decimal units the new token will use',
+      default: 18,
+    })
+}
+
+exports.task = async ({
+  web3,
+  reporter,
+  tokenName,
+  symbol,
+  transferEnabled,
+  decimalUnits,
+  silent,
+  debug,
+}) => {
+  // Decode sender
+  const accounts = await web3.eth.getAccounts()
+  const from = accounts[0]
+
+  return new TaskList(
+    [
+      {
+        title: 'Deploy the MiniMeTokenFactory contract',
+        task: async (ctx, task) => {
+          let artifact = getContract('@aragon/os', 'MiniMeTokenFactory')
```

Comment:
`@aragon/os@4.0.0` shouldn't have this contract anymore... you probably want [`@aragon/apps-shared-minime`](https://www.npmjs.com/package/@aragon/apps-shared-minime)

Reply:
Should now be fixed with `@aragon/apps-shared-minime@1.0.1` :).